recipe/meta.yaml

```
@@ -27,20 +27,60 @@ requirements:
     - rsa >=3.1.4,<5
     # extras: keep them in run to avoid breaking environments
     - aiohttp >=3.6.2,<4.0.0dev
-    - requests >=2.20.0,<3.0.0dev
+    - requests >=2.20.0,<3.0.0.dev
```

```suggestion
    - requests >=2.20.0,<3.0.0.dev0
```
https://github.com/googleapis/google-auth-library-python/blob/v2.38.0/setup.py#L31

---

done